lib/tizen_plugins.dart

```diff
@@ -117,6 +121,38 @@ mixin DartPluginRegistry on FlutterCommand {
   String get targetFile => _targetFile ?? super.targetFile;
 }
 
+/// Finds entry point functions annotated with `@pragma('vm:entry-point')`
+/// from [dartFile] and returns their names.
+List<String> _findDartEntrypoints(File dartFile) {
+  final String path = dartFile.absolute.path;
+  final AnalysisContextCollection collection =
+      AnalysisContextCollection(includedPaths: <String>[path]);
+  final AnalysisContext context = collection.contextFor(path);
+  final SomeParsedUnitResult parsed =
+      context.currentSession.getParsedUnit2(path);
```

Comment:
Why is the deprecated method `getParsedUnit2` used?

Reply:
It's not deprecated as of analyzer 2.0.0 which is currently being used.

https://pub.dev/documentation/analyzer/2.0.0/dart_analysis_session/AnalysisSession/getParsedUnit2.html